stko/molecular/topology_extractor/topology_extractor.py

```diff
@@ -0,0 +1,73 @@
+"""
+Topology Extractor
+==================
+
+Class for defining a topology from a molecule and disconnections.
+
+"""
+
+import logging
+
+from ..networkx import Network
+
+logger = logging.getLogger(__name__)
+
+
+class TopologyExtractor:
+    """
+    Extractor of topology definitions from a molecule.
+
+    """
+
+    def extract_topology(
+        self,
+        molecule,
+        atom_ids_to_disconnect,
+    ):
+
+        flat_tuple = []
+        for i in atom_ids_to_disconnect:
+            flat_tuple.append(i[0])
+            flat_tuple.append(i[1])
```

Comment:
why are the atom ids a tuple?

EDIT: Going back to the other file I can see why, but I'm gonna leave this comment here because I think the implementation is confusing (i.e. the implementation of the class is coupled to the code in the extraction example, rather than being a standalone unit)

Reply:
Fixed!